.github/workflows/ci-testing-deploy.yml

```diff
@@ -1328,7 +1328,7 @@ jobs:
   unit-test-service-library:
     needs: changes
     if: ${{ needs.changes.outputs.service-library == 'true' || github.event_name == 'push' }}
-    timeout-minutes: 18 # if this timeout gets too small, then split the tests
+    timeout-minutes: 10 # if this timeout gets too small, then split the tests
```

Comment:
why is this specific one now down to 10? what about the others?

Reply:
I have issues with the tests in the CI, I don't get it why it hangs, while locally I cannot reproduce it. I will put it back to it's original value when done.